devtools/create_project/codeblocks/create_project.cbp

```diff
@@ -36,6 +36,8 @@
 			<Add option="-Wall" />
 			<Add option="-fexceptions" />
 		</Compiler>
+		<Unit filename="..\cmake.cpp" />
+		<Unit filename="..\cmake.h" />
```

Comment:
How is CMake being used here? Why did codeblocks need to be updated?

Reply:
I simply made it consistent with the MSVC project files, but maybe it would be better in a separate PR.